pyproject.toml

```
@@ -0,0 +1,17 @@
+[tool.ruff]
+target-version = "py38"
+line-length = 120
```

We use black's default of 88
```suggestion
line-length = 88
```

---

Right, but as you check only for `F` and no `E` issues, long lines are skipped already :)
So this change has no effect and to be clear, it did not also have now...
just with enabling check for long lines, there are a few tens or hundreds issue :rabbit: